### ISSUE TITLE
Feature Request: @PlaylistFull 

### ISSUE THREAD

**Issue by pesky-wobbit** [Lyynix/MoreJournalEnrichers#21]
Lyynix, this is absolutely amazing! I can now run a session from a single journal page.

I notice Soundboard playlists don't play. That's fine, because it's not helpful to play soundboard sounds sequentially.  But, it would be really nice to have a @PlaylistFull for soundboard playlists. Now I can run my NPC and its 5 associated sounds from a journal page containing its description, actor stat-block, whispered chat, likely scenes, etc!

REQUESTS
@PlaylistFull to allow playing of individual sounds
syntax: allow menus to be separated by ";" only (without trailing space). If this isn't possible, please mention this in the guide.

minor ISSUES
@RolltableMenu says "Playlist Menu"
@RolltableFull sometimes shows entries in random order.

**Comment by Lyynix** [comment]
This is a great request, I actually don't remember why I didn't add a @PlaylistFull already

for the other request: So you want to write a Menu like this `@SceneMenu[aBDfgd32gdFGgHT3;aBDfgd32gdFGgHT3]` rather than having a space after the semicolon like this `@SceneMenu[aBDfgd32gdFGgHT3; aBDfgd32gdFGgHT3]`?
That should be doable, i think ill just make the whitespace optional

I'm getting back to work on this project in a few weeks, since I'm currently in my exam phase

**Comment by pesky-wobbit** [comment]
That's correct on both accounts. Thank you. Great work - and sorry to be so nitpicky about the scene menu syntax!

**Comment by Lyynix** [comment]
Hey, no worries
I think there is no real reason not to implement a feature request if there are no conflicts with other features :)

I'm always trying to help all users of the module

**Comment by Lyynix** [comment]
Hey, so I looked into the minor issues, and i fixed the title of @RolltableMenu

However, I can't seem to reproduce the @RolltableFull showing entries in a random order. Can you give an example of a Rolltable where this is happening?

**Comment by lucaspicerni** [comment]
Hello! Excellent module, thank you very much! Regarding individual sounds, such as sound effects, may I also suggest adding the option to preload the sound? Thank you!
